### ISSUE TITLE
[QA] Cannot send asset with Cosmos Gateway Manual route

### ISSUE THREAD

**Issue by tsadovska** [wormhole-foundation/wormhole-connect#2047]
## Environment
Checked on https://wormhole-connect-mainnet.netlify.app/ with macOS Sonoma 14.4.1 and Google Chrome Version 124.0.6367.119 (Official Build) (arm64). 

```
{
  env: 'mainnet',
  "bridgeDefaults": {
    fromNetwork: 'injective',
    toNetwork: 'bsc',
    token: 'WBNB',
  }
}
```
## Describe the current behavior in detail
Steps:
1. Open [mainnet](https://wormhole-connect-mainnet.netlify.app/?config=N4KABGCmB2BuBcYDkBbAhgS2tSAXJANOGAEQBGAThgCYDmkAIpAGZoCuANrgM4mKgQIzCgHsUAOTwB3ERQDWiJFgBWkAMa4MsSIWIRcIybhnzFZbmt2CwBuTEUB1AELinVsAF8QHoA%3D%3D)
2. Enter the Amount: 0.001 
3. Click on "Approve and proceed with transaction" button

Actual result: Transaction fails.
```
Wormhole Connect: error completing transfer TypeError: Cannot read properties of undefined (reading 'isBuffer')
    at v7 (cosmos-_oPgH_Rj.js:7:23014)
    at os (cosmos-_oPgH_Rj.js:7:22939)
    at wE (cosmos-_oPgH_Rj.js:112:137394)
    at bE (cosmos-_oPgH_Rj.js:112:137565)
    at RE.signTransaction (cosmos-_oPgH_Rj.js:112:173942)
    at async RE.signAndSendTransaction (cosmos-_oPgH_Rj.js:112:174435)
    at async WE (cosmos-_oPgH_Rj.js:112:175278)
    at async signAndSendTransaction (main-Bp7nhLg6.js:489:58597)
    at async Operator.send (main-Bp7nhLg6.js:492:131763)
    at async Object.tr [as onClick] (main-Bp7nhLg6.js:527:34268)
```
Expected result: Transaction proceeds without errors. 

## Notes/ Attachments

Issue is also reproducible when:
* Sending WMATIC from Injective (Leap) to Avalanche (MetaMask)
* Sending CELO from Evmos (Leap) to Fantom (MetaMask)

![image](https://github.com/wormhole-foundation/wormhole-connect/assets/10746654/09d68049-f0a1-4594-8b37-561c1d473f05)

**Comment by artursapek** [comment]
probably caused by SDKv2 integration. looking

**Comment by artursapek** [comment]
fix merged in #2053 please re-test

**Comment by tsadovska** [comment]
Tested OK. Please use this [link](https://docs.google.com/spreadsheets/d/128zHVbllDoayKv9srzk9tIZSlYiIuCYmGKYNqQH16zk/edit#gid=2060564317) to view the results. 

Results of testing on https://deploy-preview-2053--wormhole-connect-mainnet.netlify.app/:

![image](https://github.com/wormhole-foundation/wormhole-connect/assets/10746654/983a20d4-53d6-4182-8ed2-8981e77c3078)

Results of testing on https://wormhole-connect-mainnet.netlify.app/:
![image](https://github.com/wormhole-foundation/wormhole-connect/assets/10746654/4ae478fc-0213-49bb-a020-f7d78f423772)


Results of testing on https://preview.portalbridge.com/bceadc8c64/:

![image](https://github.com/wormhole-foundation/wormhole-connect/assets/10746654/51282e51-7b07-4b17-be50-013c8ca39363)
